### ISSUE TITLE
Applying migrations with CountryField fails with Django 3.2

### ISSUE THREAD

**Issue by SFr682k** [SmileyChris/django-countries#338]
I've created a simple model with a `CountryField`.  Creating a migration for this model works fine (at least when using the latest version from Git).  However, applying the migration fails (traceback included below).

**Environment:**
- Django 3.2b1 (from PyPI)
- django-countries 7.1.dev0 (latest version from Git)

**Traceback (most recent call last):**
```
  File "manage.py", line 22, in <module>
    main()
  File "manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "venv/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "venv/lib/python3.8/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "venv/lib/python3.8/site-packages/django/core/management/base.py", line 89, in wrapped
    res = handle_func(*args, **kwargs)
  File "venv/lib/python3.8/site-packages/django/core/management/commands/migrate.py", line 75, in handle
    self.check(databases=[database])
  File "venv/lib/python3.8/site-packages/django/core/management/base.py", line 419, in check
    all_issues = checks.run_checks(
  File "venv/lib/python3.8/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "venv/lib/python3.8/site-packages/django/core/checks/model_checks.py", line 34, in check_all_models
    errors.extend(model.check(**kwargs))
  File "venv/lib/python3.8/site-packages/django/db/models/base.py", line 1271, in check
    *cls._check_fields(**kwargs),
  File "venv/lib/python3.8/site-packages/django/db/models/base.py", line 1380, in _check_fields
    errors.extend(field.check(**kwargs))
  File "venv/lib/python3.8/site-packages/django_countries/fields.py", line 285, in check
    errors = super().check(**kwargs)
  File "venv/lib/python3.8/site-packages/django/db/models/fields/__init__.py", line 1013, in check
    *self._check_db_collation(databases),
  File "venv/lib/python3.8/site-packages/django/db/models/fields/__init__.py", line 1045, in _check_db_collation
    self.db_collation is None or
AttributeError: 'CountryField' object has no attribute 'db_collation'
```

**Comment by hugorodgerbrown** [comment]
Just came across this ourselves. The issue is with the way in which the `CountryField` initialiser is called:

```python
class CountryField(CharField):
    ...
    def __init__(self, *args, **kwargs):
        ...
        super(CharField, self).__init__(*args, **kwargs)
```

If this was changed to `super()` it would work. There are a number of instances of this that need updating. I will push a PR in due course.

**Comment by hugorodgerbrown** [comment]
See #339

**Comment by SmileyChris** [comment]
Released as 7.1 just now
